internal/pkg/service/templates/api/gen/http/templates/server/encode_decode.go

```diff
@@ -68,7 +68,7 @@ func DecodeRepositoriesIndexRequest(mux goahttp.Muxer, decoder func(*http.Reques
 		)
 		storageAPIToken = r.Header.Get("X-StorageApi-Token")
 		if storageAPIToken == "" {
-			err = goa.MergeErrors(err, goa.MissingFieldError("storageApiToken", "header"))
+			err = goa.MergeErrors(err, goa.MissingFieldError("X-StorageApi-Token", "header"))
 		}
```

Comment:
Re-generated code, off-topic, I forgot to re-generate the Templates API after the fix.